.github/workflows/quaterly-updates.yml

```diff
@@ -18,34 +18,39 @@ jobs:
       AWS_SECRET_ACCESS_KEY: ${{ secrets.DO_SECRET_ACCESS_KEY }}
       AWS_S3_BUCKET: edm-recipes
     strategy:
+      fail-fast: false
       matrix:
         dataset:
           - dcp_atomicpolygons
           - dcp_boroboundaries
           - dcp_boroboundaries_wi
+          - dcp_cb2010
+          - dcp_cb2010_wi
+          - dcp_cb2020
+          - dcp_cb2020_wi
           - dcp_cdboundaries
           - dcp_cdboundaries_wi
+          - dcp_cdta2020
+          - dcp_congressionaldistricts
+          - dcp_councildistricts
+          - dcp_councildistricts_wi
           - dcp_ct2010
           - dcp_ct2010_wi
           - dcp_ct2020
           - dcp_ct2020_wi
-          - dcp_cb2010
-          - dcp_cb2010_wi
-          - dcp_cb2020
-          - dcp_cb2020_wi
-          - dcp_cdta2020
-          - dcp_nta2020
-          - dcp_nta2010
           - dcp_electiondistricts
           - dcp_electiondistricts_wi
-          - dcp_councildistricts
-          - dcp_councildistricts_wi
           - dcp_firecompanies
           - dcp_healthareas
           - dcp_healthcenters
+          - dcp_municipalcourtdistricts
+          - dcp_nta2010
+          - dcp_nta2020
           - dcp_pad
           - dcp_policeprecincts
           - dcp_school_districts
+          - dcp_stateassemblydistricts
+          - dcp_statesenatedistricts
```

Comment:
3 added - the two state districts and municipalcourtdistricts, the rest just re-ordered alphabetically. Also added the fast-fail false

https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstrategyfail-fast

Reply:
Added fail fast because dcp_pad was failing but still wanted others to load ideally. Haven't looked into pad issue